src/app/learn/get-started/basics/build-publish/index.html

```diff
@@ -0,0 +1,43 @@
+<stache
+  navOrder="7"
+  pageTitle="Build and Publish a SPA"
+  navTitle="Build and publish">
+
+  <stache-page-summary>
+    <p>
+      After you update the SKY UX CLI with the latest version, you can build and publish your SPA. 
```

Comment:
Some of the steps in the basics section make sense to be done in order while the last two, Update CLI and Update Components, are more general and can be done at any time.  As such, I would suggest we 

[1] replace the "After you update..." part of this sentence with something like "When you're ready to build and publish your SPA." and 

[2] Move this to be directly after the "Navigation" section.